packages/mc-scripts/src/config/create-webpack-config-for-production.ts

```diff
@@ -91,7 +91,6 @@ function createWebpackConfigForProduction(
             },
             compress: {
               ecma: 5,
-              warnings: false,
```

Comment:
This property has been removed ([reference](https://github.com/terser/terser/commit/71dc3456be968ddce2281b93beb4079e609e28ef))